### PR TITLE
Replace all tracking with `quepasa` and identify

### PR DESCRIPTION
**Pull request by hedvigoscar**
with memberId

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+10.11.0
```
Comment (palmfjord) [inline .nvmrc]: ooh nice :D

**Comment by hedvigoscar** [comment]
In test environments or while running unit tests? Because I think both should be mostly fixed, right? 🤔

**Comment by palmfjord** [comment]
@hedvigoscar haha yes

**Comment by hedvigoscar** [comment]
🙁

**Comment by hedvigoscar** [comment]
@palmfjord ping 🙂

File: src/utils/tracking.ts [hunk 2]
```diff
@@ -14,35 +11,26 @@ export interface UtmParams {
...
-    eventName,
-    merge(properties || {}, { context: { ...getUtmParamsFromCookie() } }),
-    options,
-  ]
-  logInDev('[ANALYTICS TRACK] ', event)
-  const castedWindow = window as any
-  if (castedWindow && castedWindow.analytics) {
-    castedWindow.analytics.track(...event)
-  }
+export enum CustomEvents {
```
Comment (palmfjord) [inline src/utils/tracking.ts]: i don't really like the naming `Custom`, can we make it something more like `ApplicationSpecific...` or so?

Comment (hedvigoscar) [reply src/utils/tracking.ts]: Sure!

**Comment by hedvigoscar** [comment]
@palmfjord 🏓

**Pull request merged**